include/dsn/cpp/rpc_stream.h

```diff
@@ -38,6 +38,7 @@
 # include <dsn/cpp/utils.h>
 # include <dsn/service_api_c.h>
 # include <dsn/cpp/auto_codes.h>
+# include <dsn/internal/rpc_message.h>
```

Comment:
this is forbidden as dev layers cannot access internal data structures in core. the layout of the structure in core is not guaranteed to be the same as the one compiled right now in the app layer.